hledger-lib/Hledger/Read/CsvReader.hs

```diff
@@ -11,17 +11,17 @@ A reader for CSV data, using an extra rules file to help interpret the data.
 -- stack haddock hledger-lib --fast --no-haddock-deps --haddock-arguments='--ignore-all-exports' --open
```

Comment:
Switching to Text seems to add noise to this file more than others, but I expect it's worth it.